core/src/com/unciv/logic/map/MapGenerator.kt

```diff
@@ -212,7 +212,7 @@ class MapGenerator(val ruleset: Ruleset) {
         if(map.mapParameters.noRuins)
             return
         val suitableTiles = map.values.filter { it.isLand && !it.getBaseTerrain().impassable }
-        val locations = chooseSpreadOutLocations(suitableTiles.size/100,
+        val locations = chooseSpreadOutLocations( (map.mapParameters.ruinsRichness.pow(2)).toInt() * suitableTiles.size/100,
```

Comment:
Mayve the parameter `ruinsRichness` might be comprehensive of the `pow(2)` and division by `100`. I don't see the reason why not to. Of course the extreme values might be rescaled according to this.

Reply:
I didn't understand what you meant to say =/

Reply:
I know it's a bit weird doing the power 2 thing, but in my head I was going along the lines of the linear slider corresponding to the distance between ruins. So as the parameter and the area covered both scale by power 2, the slider is directly proportional to average distance between ruins